pkg/router/template/router.go

```diff
@@ -540,6 +551,30 @@ func (r *templateRouter) writeConfig() error {
 	return nil
 }
 
+// Check that the number of endpoints don't exceed the ARP cache limit
+func (r *templateRouter) checkEndpointCount() error {
+	data, err := ioutil.ReadFile("/proc/sys/net/ipv4/neigh/default/gc_thresh3")
+	if err != nil {
+		glog.Warningf("Error reading ARP neighbour information: %s", err)
+		r.arpCacheReadable = false
+		return err
```

Comment:
We already logged the warning, returning err here will log the warning one more time.